### ISSUE TITLE
Switch example crashes on Android at startup

### ISSUE THREAD

**Issue by t-arn** [beeware/toga#1163]
**Describe the bug**
Built the switch example for Android.
To overcome the bug #1162, I changed the package name to org.beeware.switches

**To Reproduce**
1. briefcase build android
2. briefcase run android
3. See error in logcat

```
2020-12-03 21:42:23.023 13711-13711/org.beeware.switch I/Python: Python app launched & stored in Android Activity class
2020-12-03 21:42:23.157 13711-13711/org.beeware.switch I/Python: [Android] Not implemented: Widget.set_hidden()
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545] JNI DETECTED ERROR IN APPLICATION: JNI GetStaticIntField called with pending exception java.lang.NullPointerException: Attempt to read from field 'int android.view.ViewGroup$LayoutParams.width' on a null object reference
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.widget.TextView.checkForRelayout() (TextView.java:8862)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.widget.TextView.setText(java.lang.CharSequence, android.widget.TextView$BufferType, boolean, int) (TextView.java:5730)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.widget.TextView.setText(java.lang.CharSequence, android.widget.TextView$BufferType) (TextView.java:5571)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.widget.TextView.setText(java.lang.CharSequence) (TextView.java:5528)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at int org.beeware.rubicon.Python.run(java.lang.String, java.lang.String[]) (Python.java:-2)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void org.beeware.android.MainActivity.startPython(java.util.Map) (MainActivity.java:224)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void org.beeware.android.MainActivity.onCreate(android.os.Bundle) (MainActivity.java:241)
2020-12-03 21:42:23.197 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.Activity.performCreate(android.os.Bundle, android.os.PersistableBundle) (Activity.java:7136)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.Activity.performCreate(android.os.Bundle) (Activity.java:7127)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.Instrumentation.callActivityOnCreate(android.app.Activity, android.os.Bundle) (Instrumentation.java:1271)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2893)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at android.app.Activity android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.app.servertransaction.PendingTransactionActions, android.content.Intent) (ActivityThread.java:3048)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.servertransaction.LaunchActivityItem.execute(android.app.ClientTransactionHandler, android.os.IBinder, android.app.servertransaction.PendingTransactionActions) (LaunchActivityItem.java:78)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.servertransaction.TransactionExecutor.executeCallbacks(android.app.servertransaction.ClientTransaction) (TransactionExecutor.java:108)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.servertransaction.TransactionExecutor.execute(android.app.servertransaction.ClientTransaction) (TransactionExecutor.java:68)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1808)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:106)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.os.Looper.loop() (Looper.java:193)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6669)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:493)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:858)
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545] 
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]     in call to GetStaticIntField
2020-12-03 21:42:23.198 13711-13711/org.beeware.switch A/.beeware.switc: java_vm_ext.cc:545]     from int org.beeware.rubicon.Python.run(java.lang.String, java.lang.String[])
```

**Expected behavior**
Example starts and works

**Environment:**
 - Operating System: Android emulator 
 - Python version: 3.8
 - Software versions:
   - Briefcase: 0.3.3
   - Toga: 0.3.0.dev25

**Comment by freakboy3742** [comment]
Thanks for the report. Ironically, this was reported against briefcase as beeware/briefcase#460 - but the logical follow up of fixing the project name here was never done.

**Comment by t-arn** [comment]
Russel, your comment probably belongs to bug #1162, not to this one here...

**Comment by freakboy3742** [comment]
Yup - definitely a mistake on my part.

**Comment by t-arn** [comment]
The issue has been solved with PR #1168
